### PR TITLE
Remove "Bind Service" feature due to the deprecation of the Service Binding Operator #4682

### PR DESCRIPTION
**Pull request by vrubezhny**
Issue: #4682

### CODE REVIEW

File: src/helm/helm.ts [hunk 1]
```diff
@@ -26,8 +26,13 @@ export type HelmRelease = {
  * @returns a list of all Helm releases in the current namespace on the current cluster
  */
 export async function getHelmReleases(): Promise<HelmRelease[]> {
-    const res = await CliChannel.getInstance().executeTool(HelmCommands.listHelmReleases(), undefined, false);
-    return JSON.parse(res.stdout) as HelmRelease[];
+    try {
+        const res = await CliChannel.getInstance().executeTool(HelmCommands.listHelmReleases(), undefined, false);
+        return JSON.parse(res.stdout) as HelmRelease[];
+    } catch (err) {
+        window.showErrorMessage(`Cannot list HELM Releases: ${err}`);
```
Comment (datho7561) [inline src/helm/helm.ts:33]: IMHO it makes more sense to let the error be passed on to whatever method is calling this one. What do you think?

File: src/oc/ocWrapper.ts [hunk 2]
```diff
@@ -85,10 +86,15 @@ export class Oc {
     }
 
     public async getAllKubernetesObjects(namespace?: string, executionContext?: ExecutionContext): Promise<KubernetesObject[]> {
-        const result = await CliChannel.getInstance().executeTool(
-            Oc.getKubernetesObjectCommand('all', namespace),
-            undefined, true, executionContext);
-        return JSON.parse(result.stdout).items;
+        try {
```
Comment (datho7561) [inline src/oc/ocWrapper.ts:89]: Same note as above here

File: src/oc/ocWrapper.ts [hunk 3]
```diff
@@ -85,10 +86,15 @@ export class Oc {
...
+            const result = await CliChannel.getInstance().executeTool(
+                Oc.getKubernetesObjectCommand('all', namespace),
+                undefined, true, executionContext);
+            return JSON.parse(result.stdout).items;
+        } catch (err) {
+            window.showErrorMessage(`Cannot list HELM Releases: ${err}`);
```
Comment (datho7561) [inline src/oc/ocWrapper.ts:95]: These are not helm releases, they are Kubernetes objects

File: src/helm/helm.ts (hunk 1 above)
Comment (vrubezhny) [reply src/helm/helm.ts:33]: When this method (and yet another one from the `osWrapper` with try-catch block added) are invoked from `OpenShiftExplorer.getChildren()` it breaks the method execution if `JSON.parse()` fails to process the `stdout` of the result - this makes the explorer tree fail on update. I could actually ignore the error (as it's usually "json is malformed"-like error, but I'd like to know when it usually happens

File: src/oc/ocWrapper.ts (hunk 2 above)
Comment (vrubezhny) [reply src/oc/ocWrapper.ts:89]: Same answer as above

File: src/oc/ocWrapper.ts (hunk 3 above)
Comment (vrubezhny) [reply src/oc/ocWrapper.ts:95]: Oh. Sure.

File: src/helm/helm.ts (hunk 1 above)
Comment (datho7561) [reply src/helm/helm.ts]: Okay sounds good!

**Review by datho7561 — APPROVED** [review]

**Pull request merged**